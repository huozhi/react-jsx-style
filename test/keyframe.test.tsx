import React from 'react'
import Keyframe from '../src/keyframe'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Keyframe identity='animation12'>
        <Keyframe.Block selector={'from'} value={{opacity: '0'}} />
        <Keyframe.Block selector={'50%'} value={{opacity: '0.5'}} />
        <Keyframe.Block selector={'to'} value={{opacity: '1'}} />
      </Keyframe>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
