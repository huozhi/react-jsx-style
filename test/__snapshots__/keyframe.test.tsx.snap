// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<style>
  @keyfame animation12 {
  from { opacity:0 }
  50% { opacity:0.5 }
  to { opacity:1 }
  }
</style>
`;
